website/snippets/_config-dbt-version-check.md

```diff
@@ -0,0 +1,6 @@
+
+Starting in 2024, when you select **Keep on latest version** in dbt Cloud, it will ignore version checking. However, if you're not using this configuration, dbt Labs recommends:
+- **Writing defensive code** &mdash; If you're developing dbt code that could be run in a variety of execution contexts (such as a package) and you depend on newer dbt functionality, you can add conditional logic that checks for the presence of other packages or macros.
+- **Pinning packages** &mdash; If you're maintaining a dbt project that installs third-party packages and are concerned about the potential for breaking changes in those packages, you should pin the package to a specific revision or `version` boundary. Since v1.7, this is the default dbt behavior, by _locking_ the version/revision of packages in development in order to guarantee predictable builds in production.
```

Comment:
I think we should reorder these bullets, and make the audience clearer for each:
- **For anyone installing packages,** we recommend...
- **For package maintainers,** we recommend writing defensive code. (I should provide an actual example of what I mean here.)

Since I wrote the first draft of this, and I have the context, I can revise this tomorrow and push up changes